css-transitions/styles.css

```diff
@@ -0,0 +1,53 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+  background-color: rgb(245, 245, 245);
+}
+
+section {
+  max-width: 992px;
+  margin: 2rem auto;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.custom-button {
+  padding: 0.5rem 1.5rem;
+  border: none;
+  border-radius: 0.25rem;
+  font-size: 1rem;
+  color: white;
+  background-color: blue;
+}
+
+.card-container {
+  text-align: center;
+}
+
+.card {
+  max-width: 18rem;
+  padding: 0.5rem;
+  margin-bottom: 2rem;
+  border: 1px solid rgb(228, 228, 228);
+  border-radius: 0.25rem;
+  background-color: white;
+  cursor: pointer;
+  transform: scale(1);
+}
+
+.card > img {
+  width: 100%;
+}
+
+.custom-button:hover {
+  background-color: red;
+  transition: 0.5s;
```

Comment:
This will only work when you start hovering on the button, it will not transition when you stop hovering. You want the transition to take effect for both.

Reply:
This can now be removed.